src/app/app.component.ts

```diff
@@ -75,31 +82,34 @@ export class AppComponent {
 @Component({
   selector: 'app-calendar-header',
   template: `
-    <mat-select placeholder="Calendar Format" [formControl]="form.controls['calendar']">
-      <mat-option *ngFor="let cal of supportedCalendars" [value]="cal">{{cal}}</mat-option>
-    </mat-select>
+        <mat-select placeholder="Calendar" [formControl]="calendar">
+          <mat-option *ngFor="let cal of supportedCalendars" [value]="cal">{{cal}}</mat-option>
+        </mat-select>
     <mat-calendar-header></mat-calendar-header>
   `,
   styleUrls: []
 })
 export class HeaderComponent<D> implements OnInit {
-  constructor(@Host() private _calendar: MatCalendar<JDNConvertibleCalendar>,
+  constructor(private _calendar: MatCalendar<JDNConvertibleCalendar>,
```

Comment:
@flavens We will have to remove `@Host` in https://github.com/dasch-swiss/knora-ui-ng-lib/blob/714826ad197eea8dd16286510e65c39b1d1a8aec/projects/knora-ui/src/lib/viewer/values/date-value/calendar-header/calendar-header.component.ts#L16

Reply:
ok

Reply:
I got it from here: https://github.com/angular/angular/issues/31539#issuecomment-511290841